dist/index.js

```diff
@@ -0,0 +1,30 @@
+"use strict";
+
+const express = require('express');
+const axios = require('axios');
+const clientID = process.env.CLIENT_ID;
+const clientSecret = process.env.CLIENT_SECRET;
+
+// const clientID = 'c820624d21ec78915700';
+// const clientSecret = 'eaae81f74c0f049528e4dc7367abb26f62c62900';
```

Comment:
we'll definitely want to delete these comments, so these credentials aren't hanging out in the repo